FlowCrypt/Functionality/Services/Compose Message Helper/ComposeMessageHelper.swift

```diff
@@ -184,7 +185,7 @@ final class ComposeMessageHelper {
     ) async throws -> [String] {
         let senderKeys = try await keyMethods.chooseSenderKeys(
             for: .encryption,
-            keys: try await appContext.keyAndPassPhraseStorage.getKeypairsWithPassPhrases(email: sender),
+            keys: appContext.keyAndPassPhraseStorage.getKeypairsWithPassPhrases(email: sender),
```

Comment:
Does it mean you use a synchronized call from now?

Reply:
No, as we have `try await` in the initial function call `try await keyMethods.chooseSenderKeys`. It's similar to the previous case when `try` was moved outside